### ISSUE TITLE
Deduplicate - median

### ISSUE THREAD

**Issue by hobovsky** [codewars/content-issues#51]
**Median**
* [All Star Code Challenge #14](https://www.codewars.com/kata/5864eb8039c5ab9cd400005c/)
  * 2 unique languages out of 6 total
  * 1 issue
  * Satisfaction: 89%
  * Solves: 997
  * Created: Dec 2016
* [Median fun fun](https://www.codewars.com/kata/582609930626631a9600003e)
  * 2 unique languages out of 6 total
  * 2 issues
  * Satisfaction: 92%
  * Solves: 778
  * Created: Nov 2016
 
Both kata have 6 languages, 7 kyu rank, and poor titles.

-------

Conclusion:
- Keep [All Star Code Challenge #14](https://www.codewars.com/kata/5864eb8039c5ab9cd400005c/)
  - [x] Add translations to languages present on the other kata
  - [x] Think of a better title
- [x] Retire [Median fun fun](https://www.codewars.com/kata/582609930626631a9600003e)

**Comment by scoraig52** [comment]
I would personally keep the first kata. It has a clear and fun description, and does not include the empty-array-check. This last part might be crucial in prd environments, but in a kata it's a maintenance hell to include details for every language how to handle empty arrays (none, nan, nil, ..).  I think such edge cases pollute a kata.

**Comment by hobovsky** [comment]
Generally I agree on the empty array part. I think we don;t really need it.

**Comment by monadius** [comment]
I also prefer the first kata. I think that the word 'median' should be added to the title of the first kata.

**Comment by Fibonaccios** [comment]
I would also keep the first one just for the description.

**Comment by hobovsky** [comment]
Could you guys suggest better title for the first kata? You can edit it into your posts, or post a new one, whatever you prefer.

**Comment by CiprianAmza** [comment]
In python the same solution passes both katas (even the names of the functions are the same). The word `median` should be added in the title, as the kata doesn't appear now if you search `median`. I assume that something like `All Star Code Challenge #14 [median]` should be enough?

**Comment by ejini6969** [comment]
I would suggest to keep the first one as updates are required on 3 languages of the second one, plus I translated the first one so I would not want my work to be wasted XD. I agree that empty arrays should not be tested and are difficult to maintain across multiple languages. I ported PHP and TS to the first one and updated TS to 4.2 version. As for the title, maybe we can name it as `All Star Code Challenge #14 (Find the median)` ???
